code/modules/events/anomaly_bluespace.dm

```diff
@@ -54,8 +54,8 @@
 					if(A.anchored) continue
 
 					var/turf/newloc = locate(A.x + x_distance, A.y + y_distance, TO.z) // calculate the new place
-					if(!A.Move(newloc)) // if the atom, for some reason, can't move, FORCE them to move! :) We try Move() first to invoke any movement-related checks the atom needs to perform after moving
-						A.loc = locate(A.x + x_distance, A.y + y_distance, TO.z)
+					if(!A.Move(newloc) && newloc) // if the atom, for some reason, can't move, FORCE them to move! :) We try Move() first to invoke any movement-related checks the atom needs to perform after moving
+						A.loc = newloc

```

Comment:
It doesn't have to be done all at once. Hell, it might even be good to wait until actual bugs are introduced. I just think making it slightly easier in the future by cutting down a few instances is a good practice for everyone to be getting into.